app/views/devise/registrations/new.html.slim

```diff
@@ -1,29 +1,79 @@
-h2 Реєстрація
-= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-  = devise_error_messages!
-  .field
-    = f.label :full_name, 'Прізвище та Ім’я'
-    br/
-    = f.text_field :full_name, autofocus: true
-  .field
-    = f.label :klass_id, 'Клас'
-    br/
-    = f.select :klass_id, options_for_select(klass_options)
-  .field
-    = f.label :email
-    br/
-    = f.email_field :email
-  .field
-    = f.label :password, 'Пароль'
-    - if @validatable
-      em
-        | (мінімум #{@minimum_password_length} символів)
-    br/
-    = f.password_field :password, autocomplete: "off"
-  .field
-    = f.label :password_confirmation, 'Підтвердження паролю'
-    br/
-    = f.password_field :password_confirmation, autocomplete: "off"
-  .actions
-    = f.submit "Реєстрація"
-= render "devise/shared/links"
+ul.nav.nav-tabs
+  li.active
+    =link_to t('.buttons.student'), "#tab1", html_options = { "data-toggle" => "tab" }
+  li
+    =link_to t('.buttons.teacher'), "#tab2", html_options = { "data-toggle" => "tab" }
+
+.tab-content
+  .tab-pane.active#tab1
+    h3
+      = t '.h_registration'
+    = form_for(resource, as: resource_name, url: registration_path(resource_name), html: {multipart: true}) do |f|
```

Comment:
it looks like you need multipart encoding only for form with file. http://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean